### PR TITLE
bettercap: 2.11 -> 2.24

### PR DESCRIPTION
**Pull request by dtzWill**
###### Motivation for this change

* 2.24: https://github.com/bettercap/bettercap/releases/tag/v2.24
* ...
* 2.11: https://github.com/bettercap/bettercap/releases/tag/v2.11

Seems to work?

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by c0bw3b** [comment]
Duplicate of #61154

**Comment by fadenb** [comment]
@c0bw3b can we re-open this PR?
It is better than the other one I created earlier.

**Comment by c0bw3b** [comment]
Sure

File: pkgs/tools/security/bettercap/default.nix [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+    owner = pname;
+    repo = pname;
     rev = "v${version}";
-    sha256 = "08hd7hk0jllfhdiky1f5pfsvl1x0bkgv1p4z9qvsksdg9a7qjznw";
+    sha256 = "1f8bzxbcyf0bpyqj6hz4l8dp5xknnb537xy9y5skcznp03i44h55";
   };
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ libpcap libnfnetlink libnetfilter_queue ];
+  buildInputs = [ libpcap libnfnetlink libnetfilter_queue libusb ];
```
Comment (c0bw3b) [inline pkgs/tools/security/bettercap/default.nix]: `libusb1` rather (I don't think it needs the compat API)

**Pull request merged**